composer.json

```diff
@@ -21,7 +21,8 @@
         "guzzlehttp/guzzle": "~6.0",
         "hylianshield/base32-crockford": "^1.0",
         "jms/serializer-bundle": "^3.0",
-        "maennchen/zipstream-php": "^2.1",
+        "laminas/laminas-diactoros": "^2.9",
+        "maennchen/zipstream-php": "dev-master",
```

Comment:
I'd feel better with a hard-coded static commit we can test against than a moving target at the bleeding edge ;-)

Reply:
At the time we run the composer add, that will be the locked version.